pkg/command/repair/res.yaml

```diff
@@ -19,13 +19,17 @@ intensity: |
   How many token ranges to repair in a single Scylla node at the same time.
   Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
   Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+  Setting intensity to a value greater than max supported results in trimming it internally.
```

Comment:
See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.